builders/integrations/relayers/gelato.md

```diff
@@ -36,278 +36,265 @@ Then, take the following steps:
 
-```
-npm install @gelatonetwork/gelato-relay-sdk
-```
+#### Payment
 
-You'll also want to install the Ethers.js library with the following command:
+Gasless transactions are _not_ actually gasless. Of course, the blockchain primitive of gas is here to stay and rightly so for sake of the underlying network's security. What Gelato Relay does is abstract the _payment_ of gas away from the end-user. Someone still pays for the gas, but now it can be done using the [payment methods](https://docs.gelato.network/developer-products/gelato-relay-sdk/payment) that Gelato Relay supports, which offer superior features and convenience over on-ramps from fiat and KYC. A user can pay Gelato during the call forward using their own smart contract logic (or use Gelato's [premade logic](https://docs.gelato.network/developer-products/gelato-relay-sdk/prerequisites#gelatos-relay-context) for easy integration). Otherwise, if they would like, they can sign a message using their wallet and send this signature using Gelato Relay SDK's [user authentication](https://docs.gelato.network/developer-products/gelato-relay-sdk/sdk-methods/relaywithsponsoreduserauthcall). Gelato's decentralised network of executors will then relay the message on-chain on the user's behalf.
 
-```
-npm install ethers
-```
+### Why use Gelato Relay as a developer? {: #why-gelato-relay }
+
+Using Gelato Relay, we relay your user's transactions on-chain (with optional secure [ERC-2771](https://docs.gelato.network/developer-products/gelato-relay-sdk/prerequisites#erc-2771)  and [EIP-712](https://docs.gelato.network/developer-products/gelato-relay-sdk/prerequisites#what-is-a-meta-transaction) support), enabling secure gasless transactions for an ultra smooth UX for your dApp. This allows for a variety of new web3 experiences, as the user can now pay by only signing a message, or their transaction costs can be sponsored by the dApp developer. As long as the gas costs are covered in one of the multiple [payment methods](https://docs.gelato.network/developer-products/gelato-relay-sdk/payment) that Gelato supports, we handle the rest reliably, quickly and securely.
 
-Next, you'll need to create a javascript file for your script. You can create a `hello-world.js` file by running:
+### Installation {: #installation }
+
+#### Gelato Relay SDK
+
+Gelato Relay SDK is an [NPM package](https://www.npmjs.com/package/@gelatonetwork/relay-sdk){target=\_blank} that can be installed locally with the following command:
 
 ```
-touch hello-world.js
+npm install @gelatonetwork/relay-sdk
 ```
 
-Now you're ready to build. First, you need to import the Gelato Relay SDK and EthersJS:
+or
 
 ```
-  import { Wallet, utils } from "ethers";
-  import { GelatoRelaySDK } from "@gelatonetwork/gelato-relay-sdk";
+yarn add @gelatonetwork/relay-sdk
 ```
```

Comment:
to make this more concise you could do: 

```
=== "NPM"
    ```
    npm install @gelatonetwork/relay-sdk
    ```

=== "Yarn"
    ```
    yarn add @gelatonetwork/relay-sdk
    ```
```

there is an example of it here: 
- https://docs.moonbeam.network/builders/build/eth-api/dev-env/foundry/#deploying-the-contract
- https://raw.githubusercontent.com/PureStake/moonbeam-docs/master/builders/build/eth-api/dev-env/foundry.md

Reply:
Same with the other packages that need to be installed